internal/feed/rss.go

```diff
@@ -325,14 +324,12 @@ func isFreeleech(str []string) bool {
 	return false
 }
 
-// readSizeFromDescription get size from description
-func readSizeFromDescription(str string) string {
-	matches := rxpSize.FindStringSubmatch(str)
-	if matches == nil {
-		return ""
+// pullSizeFromDescription get size from description
+func pullSizeFromDescription(str string, r *domain.Release) {
```

Comment:
Minor nitpick but I think the old name `readSizeFromDescription` is better, or `parse` instead of `read`. we don't use the `pull` terminology anywhere else.